react/admin/ReturnsSettings.tsx

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable @typescript-eslint/explicit-member-accessibility */
-import React, { Component } from 'react'
+import { Component, Fragment } from 'react'
```

Comment:
Any reason to remove React from context here? I think we need to keep it since we have JSX in the file and we are using React v16. Eslint should got this, but I think there is some miss configuration here.